test/unit/presenters/formats/edition_format_presenter_test.rb

```diff
@@ -171,5 +172,15 @@ def result
         refute result.has_key?(:access_limited)
       end
     end
+
+    should "[:format]" do
+      edition.expects(:format).returns('foo')
+      assert_equal 'frontend', result[:rendering_app]
+    end
+
+    should "[:format_gf]" do
```

Comment:
the convention in this test class is to use `context` blocks to identify behaviours based on alternative data scenarios.  However, this case should probably be in the `answer_presenter_test` class under `rendering_app`